### PR TITLE
docs: Add playbook on practices

### PR DESCRIPTION
**Pull request by ansor4**
This is a first draft at a playbook for practices at Artsy! I've whipped this up on a late Friday afternoon so please leave any feedback on structure, content, if I'm missing links, etc!

### CODE REVIEW

File: practices/playbook.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+- Change the board settings to filter by label and remove project filter
+
+The board should be able to show tickets from other projects but with the practice label.
+
+See the Platform Practice's
+[Kanban Board](https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=79&projectKey=PLATFORM)
+
+## Running The Practice Meeting
+
+The facilitator of the practice meeting should go over the agenda. Afterwards,whoever is championing the topic at
```
Comment (damassi) [inline practices/playbook.md:69]:
```suggestion
The facilitator of the practice meeting should go over the agenda. Afterwards, whoever is championing the topic at
```

**Review by damassi — APPROVED** [review]
This is really helpful @ansor4 👍

File: practices/playbook.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+[Discussion Topic Playbook](https://www.notion.so/artsy/Discussion-Topic-Playbook-4a11a9c2313c409782b3f33eda9bb4ee)
+
+### Create Jira Board To Track Action Items
+
+Action items from the practice meeting should be tracked on product teams' projects but visible in one aggregated
+board. The practice should not have a separate board to track work. Action items from practices should ideally be
+folded into a normal sprint on a product team.
+
+- Create a new board
+- Create a new label to mark tickets as related to the practice
```
Comment (MounirDhahri) [inline practices/playbook.md:59]: 👏

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
Thanks @ansor4 🙏

**Comment by alexpopa-artsy** [comment]
LGTM! Thank you @ansor4!
Tagging @anipetrov for visibility

**Review by alexpopa-artsy — APPROVED** [review]

**Review by dblandin — APPROVED** [review]
Love this! 💜

File: practices/README.md [hunk 3]
```diff
@@ -16,6 +16,7 @@ is useful for deciding how to organize and divide large efforts up among product
 | [The history of practices at Artsy](/practices/history.md#readme) | How did we get to where we are |
 | [The Mobile Practice](/practices/mobile.md#readme) | How does Artsy make mobile Software? |
 | [The Platform Practice](/practices/platform.md#readme) | What do folks with a platform focus do? |
+| [The Practice Playbook](/practices/playbook.md#readme) | How do you run a practice at Artsy? |
```
Comment (joeyAghion) [inline practices/README.md:19]: I wonder if `playbooks/practices.md` might be a better location than `practices/playbook.md`, so it's grouped with other playbooks.

File: practices/playbook.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+3. Type of discussion: tech plan review, brainstorm, knowledge share, etc
+4. Goal of discussion: what questions should be answered? what kind of feedback does the champion hope to get?
+
+See the Platform Practice's
+[Discussion Topic Playbook](https://www.notion.so/artsy/Discussion-Topic-Playbook-4a11a9c2313c409782b3f33eda9bb4ee)
+
+### Create Jira Board To Track Action Items
+
+Action items from the practice meeting should be tracked on product teams' projects but visible in one aggregated
+board. The practice should not have a separate board to track work. Action items from practices should ideally be
```
Comment (joeyAghion) [inline practices/playbook.md:55]: "The practice should not have a separate board to track work" is confusing in relation to "create a new board" below. Maybe this should say "...should not have a separate project to track work."

File: practices/playbook.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+folded into a normal sprint on a product team.
+
+- Create a new board
+- Create a new label to mark tickets as related to the practice
+- Change the board settings to filter by label and remove project filter
```
Comment (joeyAghion) [inline practices/playbook.md:60]: I struggled with these steps a bit ([here](https://artsy.slack.com/archives/C8VH98J6Q/p1572622567002000)). You have to label an issue with the new practice label first, then create a "saved filter query" targeting that label, then create a new board in an existing project (choose the most relevant) and associate it with the saved filter query.

**Review by joeyAghion — APPROVED** [review]

File: practices/playbook.md (hunk 5 above)
Comment (ansor4) [reply practices/playbook.md:60]: thanks! i'll try to fold those steps in clearly

**Pull request merged**